core/src/gateway_createfromjson.c

```diff
@@ -129,49 +131,72 @@ GATEWAY_HANDLE Gateway_CreateFromJson(const char* file_path)
     return gw;
 }
 
+static void rollbackModules(size_t success_modules_entries_count, GATEWAY_HANDLE gw, VECTOR_HANDLE modules_added_successfully)
```

Comment:
Good one. Done.

---
In reply to: [109018027](https://github.com/Azure/azure-iot-gateway-sdk/pull/189#discussion_r109018027) [](ancestors = 109018027)